presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1229,6 +1230,11 @@ public SystemSessionProperties(
                         HASH_BASED_DISTINCT_LIMIT_ENABLED,
                         "Hash based distinct limit enabled",
                         featuresConfig.isHashBasedDistinctLimitEnabled(),
+                        false),
+                integerProperty(
+                        MAX_STAGE_COUNT_FOR_EAGER_SCHEDULING,
```

Comment:
It make more sense to call this config Adaptive scheduling over eager scheduling. wdyt?

Reply:
Hmm, I'm putting "max for eager scheduling", meaning the maximum level before which we use eager scheduling, and after which we use phased scheduling.